src/components/body/BodyMainPanel.tsx

```diff
@@ -45,20 +57,54 @@ const BodyMainPanel: React.FC<IProps> = ({
 
   const contentContainerStyle = React.useMemo(
     () => ({
-      paddingLeft: styleLeft,
-      paddingTop: styleTop,
+      top: styleTop,
+      left: styleLeft,
+      paddingLeft: _scrollLeft,
+      paddingTop: _scrollTop,
       height: dataLength * bodyRowHeight,
       width: bodyContentWidth,
     }),
-    [styleTop, styleLeft, dataLength, bodyRowHeight, bodyContentWidth]
+    [
+      styleTop,
+      styleLeft,
+      _scrollTop,
+      _scrollLeft,
+      dataLength,
+      bodyRowHeight,
+      bodyContentWidth,
+    ]
   );
 
+  const onScroll: React.UIEventHandler<HTMLDivElement> = throttle(
+    React.useCallback(() => {
+      const scrollTop = panelScrollRef.current?.scrollTop || 0;
+      const scrollLeft = panelScrollRef.current?.scrollLeft || 0;
```

Comment:
In phone, the scrollTop and scrollLeft values could be negative or over the container div.
Please add some logic for scroll value problems.